website/source/docs/vagrant-cloud/api.html.md.erb

```diff
@@ -530,6 +530,101 @@ Sends a 2FA code to the requested delivery method.
 }
 ```
 
+## Search
+
+### Search for boxes
+
+`GET /api/v1/search`
+
+#### Arguments
+
+* `q` - (Optional) The search query. Results will match the `username`, `name`, or `short_description` fields for a box. If omitted, the top boxes based on `sort` and `order` will be returned (defaults to "downloads desc").
+* `provider` - (Optional) Filter results to boxes supporting for a specific provider.
+* `sort` - (Optional, default: `"downloads"`) The field to sort results on. Can be one of `"downloads"`, `"created"`, or `"updated"`.
```

Comment:
Maybe this is a better question for the corresponding website PR, but what gets returned when you provide a sort key that isn't supported? Is there a new HTTP response code that needs to be documented here?

Reply:
I think right now it just ignores it, but I will check.

Reply:
Validated now! Will return 400.